code/game/turfs/simulated/walls.dm

```diff
@@ -159,7 +161,7 @@
 
 	return
 
-/turf/simulated/wall/fire_act(datum/gas_mixture/air, exposed_temperature, exposed_volume)//Doesn't fucking work because walls don't interact with air :(
+/turf/simulated/wall/fire_act(datum/gas_mixture/air, exposed_temperature, exposed_volume) //Doesn't fucking work because walls don't interact with air :[
```

Comment:
Really? :P

Reply:
loose brackets in code makes rainbow brackets die so i had to do it to em